addons/l10n_sa_edi/models/account_edi_xml_ubl_21_zatca.py

```diff
@@ -236,13 +236,8 @@ def _l10n_sa_get_monetary_vals(self, invoice, vals):
         allowance_charge_vals = vals['vals']['allowance_charge_vals']
         allowance_total_amount = sum(a['amount'] for a in allowance_charge_vals if a['charge_indicator'] == 'false')
         if downpayment_vals:
-            # - BR-KSA-80: To calculate prepaid, we need to sum up the amounts of standard lines (neither a down-payments, nor a discount)
-            #   then we add the total amount of the down-payment.
-            # - BR-CO-16: To calculate payable amount, we substract the calculated prepaid amount from the total tax inclusive amount of the invoice
-            regular_line_vals = invoice._prepare_edi_tax_details(
-                filter_invl_to_apply=lambda line: (line.price_subtotal > 0 and not line._get_downpayment_lines())
-            )
-            prepaid_amount = abs(regular_line_vals['base_amount_currency'] + regular_line_vals['tax_amount_currency']) + downpayment_vals['total_amount']
+            # - BR-KSA-80: To calculate prepaid, we deduct prepaid amount from total tax inclusive amount
```

Comment:
do you mean "To calculate payable?"

Reply:
@jco-odoo absolutely, good catch. Fixed it :+1: